packages/web-react/src/components/Toast/README.md

```diff
@@ -263,13 +264,108 @@ const [isOpen, setIsOpen] = React.useState(false)
 </Toast>
 ```
 
-[web-toast]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/src/scss/components/Toast
-[mdn-role-log]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/log_role
-[mdn-aria-live]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live
+## UncontrolledToast
+
+```jsx
+import { ToastProvider, UncontrolledToast } from '@lmc-eu/spirit-web-react/components';
+```
+
+### Minimal props
+
+```jsx
+const { showToast } = useUncontrolledToast(); // must be inside ToastProvider
+
+<ToastProvider>
+  <Button type="button" onClick={() => showToast(ToastTextWithLink, 'toast-id')}>
+    Show Toast
+  </Button>
+
+  <UncontrolledToast />
+</ToastProvider>;
+```
+
+### All possible props
+
+```jsx
+const { showToast } = useUncontrolledToast(); // must be inside ToastProvider
+
+<ToastProvider>
+  <Button
+    type="button"
+    onClick={() => showToast('Toast message', 'toast-id', {
+      autoClose: true
+      color: 'danger'
+      iconName: 'download'
+    })}
+  >
+    Show Toast
+  </Button>
+
+  <UncontrolledToast alignmentX="right" alignmentY="top" closeLabel="Close toast" hasIcon isDismissible />
```

Comment:
message?

Reply:
message added

Reply:
Reverting back, I just realized there should not be any message because the message is dynamic and you are setting it with the call function. So the message is added to this component from context.